recipes/epoxy/build.sh

```diff
@@ -0,0 +1,27 @@
+#! /bin/bash
+# Prior to conda-forge, Copyright 2017-2019 Peter Williams and collaborators.
+# This file is licensed under a 3-clause BSD license; see LICENSE.txt.
+
+set -ex
+
+if [ -n "$OSX_ARCH" ] ; then
```

Comment:
FWIW, in other staged-recipe PRs, I've seen the conda-forge core team ask people to use the following method to detect OSX.  I don't know if they will request it here, too.

```bash
if [[ $target_platform == osx* ]]
```

Reply:
That's a good suggestion.

Reply:
Oooh, I do like that better.